.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Please share the command, input, and output that can be used to reproduce the bug.
+
+Note: Please make sure to redact sensitive information from your policies, ARNs, and account IDs.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Environment (please complete the following information):**
+ - OS: [e.g., macOS, Linux, Windows]
+ - IAM Policy Autopilot Version: [e.g., 0.1.0]
```

Comment:
Should we add instructions for getting the version here? Might not be obvious for someone running the mcp-server with `@latest`